### ISSUE TITLE
Unexpected discrepancy between renders of code-blocks and code-span

### ISSUE THREAD

**Issue by ashmaroli** [gettalong/kramdown#752]
Hi, the following discrepancy was detected while investigating https://github.com/jekyll/jekyll/issues/8990

For context, Jekyll sets `config["syntax_highlighter_opts"]["default_lang"] ||= "plaintext"` prior to passing `config` onto kramdown.
With this set up, when Jekyll renders the following snippet:
~~~
This is a Ruby code fragment `x = Class.new`{:.language-ruby}

```
def what?
  42
end
```
{: .language-ruby}
~~~
into:
```
<p>This is a Ruby code fragment <code class="language-plaintext highlighter-rouge"><span class="n">x</span> <span class="o">=</span> <span class="no">Class</span><span class="p">.</span><span class="nf">new</span></code></p>

<div class="language-ruby highlighter-rouge"><div class="highlight"><pre class="highlight"><code><span class="k">def</span> <span class="nf">what?</span>
  <span class="mi">42</span>
<span class="k">end</span>
</code></pre></div></div>
```

Note how code-span gets `language-plaintext` assigned while the code-block gets `language-ruby`.

I'm not saying this is a bug with kramdown. But would like to get your opinion on the situation.

**Comment by gettalong** [comment]
Hi @ashmaroli!

I can confirm that this is a bug in the kramdown HTML converter.

**Comment by ashmaroli** [comment]
Thank you for looking into this, @gettalong :)

**Comment by gettalong** [comment]
@ashmaroli Fix will be released in a bit.

**Comment by ashmaroli** [comment]
Thanks for the quick fix, @gettalong
i confirm that **this issue has been resolved with kramdown-2.4.0**

But I don't see the change on the `master` branch nor a `REL_2_4_0` tag pushed to this repo..

**Comment by gettalong** [comment]
@ashmaroli Thanks, fixed!
